src/consensus/pbft_manager.cpp

```diff
@@ -659,8 +661,6 @@ bool PbftManager::stateOperations_() {
   LOG(log_tr_) << "PBFT current round is " << round;
   LOG(log_tr_) << "PBFT current step is " << step_;
 
-  if (is_syncing_()) return true;
```

Comment:
so what problems did this cause ?

Reply:
@JakubFornadel  it causes the node to have trouble from recovering from a pbft state out of sync caused sync request.    We should add a check like this back in once we differentiate from bootstrapping and block syncing from pbft syncing.